src/index.html

```diff
@@ -4,43 +4,73 @@
           </label>
         </div>
-      -->
+
+
+        <select class="p-2 mb-2 mt-2 bg-gray-800 text-white focus:outline-none hover:bg-gray-700 transition-all duration-500 rounded" id="qualitySelector">
+          <option value="highest">Highest</option>
+          <option value="lowest">Lowest</option>
+          <option value="highestaudio">Highest Audio</option>
+          <option value="lowestaudio">Lowest Audio</option>
+          <option value="highestvideo">Highest Video</option>
+          <option value="lowestvideo">Lowest Video</option>
+        </select>
+        
+        <select class="p-2 mb-2 mt-2 bg-gray-800 text-white focus:outline-none hover:bg-gray-700 transition-all duration-500 rounded" id="exportTypeSelector">
+          <option value="mp4">MP4</option>
+        </select>
+        
+
         <br />
-        <button class="p-2 bg-black text-white focus:outline-none hover:bg-gray-700 transition-all duration-500" onclick="downloadVideo()">Download</button>
+        <div class="d-flex">
+          <button class="p-2 bg-blue-600 text-white focus:outline-none hover:bg-blue-700 transition-all duration-500 rounded" onclick="downloadVideoQueue()">Download</button>
+          <button class="p-2 bg-red-600 text-white focus:outline-none hover:bg-red-700 transition-all duration-500 rounded" onclick="cleanQueue()">Clean Queue</button>
+        </div>
         <br />
-        <p class="font-bold">Video download progress: <span class="font-mono text-black bg-gray-200 p-1" id="downloadProgressTwo">0%</span></p>
-        <div class="text-black bg-gray-200 mx-auto h-2.5 dark:bg-gray-700 m-2">
-          <div id="downloadProgress" class="bg-blue-600 h-2.5" style="width: 0%"></div>
+        <div style="opacity: 0;display: none;" id="downloadProgressDiv">
+          <p class="font-bold">Video download progress: <span class="font-mono text-black bg-gray-200 p-1" id="downloadProgressTwo">0%</span></p>
+          <div class="text-black bg-gray-200 mx-auto h-2.5 dark:bg-gray-700 m-2 rounded-3xl overflow-hidden">
+            <div id="downloadProgress" class="transition-all bg-blue-600 h-2.5" style="width: 0%"></div>
+          </div>
+          <ins onclick="openVideosFolder()" class="hover:underline cursor-pointer text-green-500 mb-2" id="done"></ins>
+          
+          <code id="info" class="text-blue-400 mt-2 mb-2"></code>
+          
+          <code id="error" class="text-rose-500 mt-2 mb-2"></code>
         </div>
-        <p class="font-bold text-white">Filename: <span class="font-mono text-black bg-gray-200 p-1" id="videoFilename"></span></p>
-        <ins onclick="openVideosFolder()" class="hover:underline cursor-pointer text-green-500" id="done"></ins>
-        <code id="info"></code>
-        <code id="error"></code>
       </div>
     </div>
 
     <script src="js/downloader.js"></script>
+
+    <script>
+      const noConnection = document.getElementById("no-connection")
```

Comment:
What's this?

Reply:
Idk i planned to add these crap, forgo to remov